tree.go

```diff
@@ -110,7 +112,30 @@ func addDumpStatsToTree(tree treeprint.Tree, dumpStats *stats_pb.DumpStatsEntry)
 	statsTree.AddBranch(fmt.Sprintf("Pages written: %d us", dumpStats.GetPagesWritten()))
 }
 
-func addPsTreeToTree(tree treeprint.Tree, psTree *crit.PsTree) {
+func addPsTreeToTree(tree treeprint.Tree, psTree *crit.PsTree) error {
+	psRoot := psTree
+	if pID != 0 {
+		// dfs performs a short-circuiting depth-first search.
+		var dfs func(*crit.PsTree) *crit.PsTree
+		dfs = func(root *crit.PsTree) *crit.PsTree {
+			if root.PID == pID {
+				return root
+			}
+			for _, child := range root.Children {
+				if ps := dfs(child); ps != nil {
+					return ps
+				}
+			}
+			return nil
+		}
+
+		ps := dfs(psTree)
+		if ps == nil {
+			return fmt.Errorf("no process with PID %d", pID)
```

Comment:
Would it make sense to extend this message to also show the available PIDs?

Reply:
It would work well for smaller checkpoints, but I'm unsure about how we can manage the output for bigger process trees. Either way, wouldn't it be trivial for the user to run `checkpointctl inspect --ps-tree` to see the available PIDs before using the PID flag?

Reply:
It makes sense. Should we add something like the following to the error message?
> Use `checkpointctl inspect --ps-tree` to list available PIDs